pages/contact.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="hero">
+      <img
+        src="/images/logo.png (2).png"
+        alt="Seven Foods And Grill Logo"
+        width="150"
+        height="auto"
+      />
+      <nav class="navbar">
+        <ul>
+          <li><a href="../pages/about.html">About</a></li>
+          <li><a href="../pages/contact.html">Contact</a></li>
+        </ul>
+        <hr />
+      </nav>
```

Comment:
Similar to the previous code, ensure that the navigation links in the header (<a> tags in the <nav>) have accurate paths.

Reply:
changes done.